### DISCUSSION TITLE
Separate coordinate and force files from CP2K for conversion

### DISCUSSION THREAD

**Discussion by nihals756** [deepmodeling/dpdata#723] [Q&A]
Hello, I have files cord.xyz, forces.xyz-1_0.xyz and an output file for each frame, obtained from CP2K. The cord.xyz file mentions the energy of the system. I tried the following for the first frame:
data = dpdata.System()
data.from_xyz('cord.xyz')

This works fine, and I see the data summary. Can I now add the force data to this System? I am also not sure if the energy has been added.

I have also tried fmt='cp2kdata/md' and 'cp2k/output'. But when the output file was read, it showed that 'PRINT_LEVEL MEDIUM' is required, and my output files had 'PRINT_LEVEL LOW', which is why I am now trying with 'from_xyz', to work with this data.

**Comment by robinzyb** [comment]
If the md simulation is NVT, just explicitly set the `cp2k_output_name=None` and  `cells=np.array([[10.0, 0,0 ], [0, 10.0, 0], [0,0,10.0]])` (Please put your cells here, this is just an example) in `dpdata.LabeledSystem("dir_name", cp2k_output_name, cells=cells, fmt='cp2kdata/md')`
